test/integration/delete_category_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+
+class DeleteCategoryTest < ActionDispatch::IntegrationTest
+
+  setup do
+    admin_user = User.create!(username: "vhodder", email: "vhodder@email.com",
+                              password: "password", admin: true)
+    sign_in_as(admin_user)
+    @category = Category.create!(id: 1, name: "Sports")
+    @article = Article.create!(id: 1, title: "Rugby is great", description: "This is a great sport", 
+                                user: admin_user, category_ids: [@category.id])
+  end
+
+
+  test "should not delete associated article" do
```

Comment:
Wir legen Wert auf eine aussagekräftige Beschreibung, von dem was wir tun. :-) Vielleicht so:
```suggestion
  test "delete a category without deleting an associated article" do
```
Dann gerne mergen!

Reply:
Viel besser, danke!